index.html

```diff
@@ -2,6 +2,8 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Fork Me! FCC: Test Suite Template</title>

```

Comment:
In index.html

- Kindly give your title a class, it will help you in styling your project.

Reply:
> In index.html
> 
> * Kindly give your title a class, it will help you in styling your project.

- [ ] You did well 😃 by using the inline comments, but as you see here the reason for the change is not clear to the student, so kindly 😊 comments should be clear and easy to understand. For Example:
* This is a clear comment: *The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?*
* This is a vague comment: *The README is not descriptive, why don't you make it better?*💯